hera_sim/io.py

```diff
@@ -29,10 +28,40 @@ def empty_uvdata(
     conjugation=None,
     **kwargs,
 ):
-    # TODO: docstring
-    """"""
+    """Create an empty UVData object with given specifications.
+
+    Parameters
+    ----------
+    Ntimes : int, optional
+        NUmber of unique times in the data object.
+    start_time : float, optional
+        Starting time (Julian date) by default 2456658.5
+    array_layout : [type], optional
+        Specify an array layout.
```

Comment:
The typing for this is `Dict[int, np.ndarray]`.

Reply:
Gotcha.